### PR TITLE
{Service Fabric} Bump sfmc sdk version

### PR DESCRIPTION
**Pull request by mwesigwaguma**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/servicefabric/_params.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('vm_size', help='The size of virtual machines in the pool. All virtual machines in a pool are the same size.', default='Standard_D2')
         c.argument('vm_image_publisher', help='The publisher of the Azure Virtual Machines Marketplace image.', default='MicrosoftWindowsServer')
         c.argument('vm_image_offer', help='The offer type of the Azure Virtual Machines Marketplace image.', default='WindowsServer')
-        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2019-Datacenter')
+        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2022-Datacenter')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/servicefabric/_params.py:326]: I would like to confirm if changing the default values will cause a breaking change in the customer's automated scripts? Can the functions of `2022-Datacenter` fully cover `2019-Datacenter`?

**Comment by zhoxing-ms** [comment]
Please note that we are launching the release for this sprint this week. Please resolve all comments by tomorrow, otherwise the release of this PR will have to be postponed to the next sprint **(on 10-08)**

File: src/azure-cli/azure/cli/command_modules/servicefabric/_params.py (hunk 1 above)
Comment (mwesigwaguma) [reply src/azure-cli/azure/cli/command_modules/servicefabric/_params.py:326]: no, no shouldn't affect customer experience

**Comment by yanzhudd** [comment]
please fix the CI issues

**Pull request closed without merging**